website/addons/base/__init__.py

```diff
@@ -864,6 +861,66 @@ def after_delete(self, node, user):
         """
         pass
 
+############
+# Archiver #
+############
+class GenericRootNode(object):
+    path = '/'
+    name = ''
+
+class StorageAddonBase(object):
```

Comment:
Should `StorageAddonBase` imply `BaseNodeSettings`?
I'd prefer to avoid multiple inheritance when possible.